miniapps/nurbs/nurbs_ex3.cpp

```diff
@@ -0,0 +1,279 @@
+//                                MFEM Example 3 -- modified for NURBS FE
+//
+// Compile with: make nurbs_ex3
+//
+// Sample runs:  nurbs_ex3 -m ../../data/square-nurbs.mesh
+//               nurbs_ex3 -m ../../data/square-nurbs.mesh -o 2
```

Comment:
There is no error convergence with increasing order. With `-o 3`: || E_h - E ||_{L^2} = 9.19727

Reply:
Will look into. There are clearly still some (minor???) issues.

Reply:
For nurbs_ex24 `-p 0` and `-p 1`  that also use the Hcurl space do converge. So hopefully this is an issue with nurbs_ex3 and not the Hcurl implementation itself.

Reply:
I found the problem. The bdr dofs where not correctly aggregated. I think I have a solution that will also fix the `Table` issue. This will take a couple days...

Reply:
I have corrected the bdr dofs  -- and the table  -- for `-o 1` the solutions converge. For other orders there still seems an issue

Reply:
There is still no convergence with increasing order.

Reply:
Thanks for fixing this. It is converging in 2D. In 3D, it is not converging. Could there still be issues with boundary conditions in 3D?

Reply:
3D did work before...  I am getting tired of myself :(
Thank you for your diligence. I am will have a look. Hopefully it is the same normal vs tangent error.

Reply:
Fixed it. I checked all the convergences.... 2D/3D on ex3(hcurl), ex5(hdiv) and ex24(both) and it all _seemed_ to work. Fingers crossed.